orttraining/orttraining/test/python/orttraining_run_glue.py

```diff
@@ -69,31 +69,45 @@ def setUp(self):
         self.logging_steps = 10
         self.rtol = 1e-02
 
+    def verify_old_and_new_api_are_equal(self, results_per_api):
+        new_api_results = results_per_api[True]
+        old_api_results = results_per_api[True]
```

Comment:
```suggestion
        old_api_results = results_per_api[False]
```
SHould this be `False`?